apps/prairielearn/tsconfig.json

```diff
@@ -3,6 +3,8 @@
     // Bun requires these to be set at the root `tsconfig.json` file.
     "jsx": "react-jsx",
     "jsxImportSource": "preact",
+    // @typescript-eslint requires this to be set at the root `tsconfig.json` file.
+    "strictNullChecks": true,
```

Comment:
It doesn't seem to understand the `references` of this file. I tried adding it to all referenced files and re-compiling, but it only seems to look at this file.

Reply:
Interesting, is there a reason this wasn't relevant until now? How did any of the previous changes work?

Reply:
This only impacts the `vite.config.ts` and `vitest.config.ts` files. It's possible I have misconfigured something else here.

Reply:
Ah - so `@typescript-eslint` _does_ understand references then? The Vite/Vitest config files don't live in either of the projects, so it seems completely expected that we'd need config in the root file, as that's the one that would apply to those config files.